vpc/vpc-meta.template

```diff
@@ -226,6 +226,30 @@
         ]
       }
     },
+    "NACLAdminGroup": {
+      "Type": "AWS::IAM::Group",
+      "Properties": {
+        "NACLAdminPolicyDocument": {
+          "Version": "2012-10-17",
+          "Statement": [
+        {
+            "Action": [
+                "ec2:CreateNetworkAclEntry",
+                "ec2:DeleteNetworkAclEntry",
+                "ec2:DescribeNetworkAcls",
+                "ec2:ReplaceNetworkAclEntry",
+                "ec2:DescribeVpcAttribute",
+                "ec2:DescribeVpcs"
+            ],
+            "Effect": "Allow",
+            "Resource": "*"
+                }
+              ]
+            }
+          }
+        ]
```

Comment:
this brace does not match, same with one curly brace:

parse error: Unmatched ']' at line 250, column 11